SpectraLogic.SpectraStorageBrokerClient/ResponseParsers/RetryResponseParser.cs

```diff
@@ -0,0 +1,40 @@
+﻿/*
+ * ******************************************************************************
+ *   Copyright 2014-2018 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+using log4net;
+using SpectraLogic.SpectraStorageBrokerClient.Model;
+using SpectraLogic.SpectraStorageBrokerClient.Runtime;
+using System.Net;
+
+namespace SpectraLogic.SpectraStorageBrokerClient.ResponseParsers
+{
+    internal class RetryResponseParser : IResponseParser<IJob>
+    {
+        #region Fields
```

Comment:
any particular reason your using the `#region` here?

Reply:
This region wraps the Fields in this class

Reply:
Why? What does wrapping the fields achieve?

Reply:
it's just a design decision in this project

Reply:
fair enough

Reply:
It allows visual studio to collapse the region down.